src/api/cloud/project/ip/cloud-project-ip.v6.service.js

```diff
@@ -1,25 +1,10 @@
-angular.module('ovh-api-services').service('OvhApiCloudProjectIpV6', ($resource, $cacheFactory) => {
-  const queryCache = $cacheFactory('OvhApiCloudProjectIpV6Query');
-  const cache = $cacheFactory('OvhApiCloudProjectIpV6');
-
-  const ips = $resource('/cloud/project/:serviceName/ip', {
-    serviceName: '@serviceName',
-  }, {
-    get: { method: 'GET', cache },
-    query: {
-      method: 'GET',
-      cache: queryCache,
-      isArray: true,
-    },
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectIoCapabilitiesStreamRegionV6', ($resource) => {
+    const resource = $resource('/cloud/project/:serviceName/io/capabilities/stream/region/:regionName', {
+      serviceName: '@serviceName',
+      regionName: '@regionName',
+    });
+
+    return resource;
   });
-
-  ips.resetCache = function () {
```

Comment:
Be careful because by removing theses method you will introduce a **Breaking Change**.

Could you please confirm that these methods are no more used?

Reply:
Oops, my bad! Didn't see that I removed that :-$
Thx